docs/tutorials/onnx/fine_tuning_gluon.md

```diff
@@ -0,0 +1,441 @@
+base_url = "https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/doc/tutorials/onnx/{}?raw=true"
+
+if not os.path.isdir(image_folder):
+    os.makedirs(image_folder)
+    for image in images:
+        wget.download(base_url.format("{}/{}.jpg".format(image_folder, image)), image_folder)
+if not os.path.isfile(utils_file):
+    wget.download(base_url.format(utils_file))
+```
+
+
+```python
+from utils import *
+```
+
+## Downloading a model from the ONNX model zoo
+
+We download a pre-trained model, in our case the [vgg16](https://arxiv.org/abs/1409.1556) model, trained on [ImageNet](http://www.image-net.org/) from the [ONNX model zoo](https://github.com/onnx/models). The model comes packaged in an archive `tar.gz` file containing an `model.onnx` model file and some sample input/output data.
+
+
+```python
+base_url = "https://s3.amazonaws.com/download.onnx/models/" 
+current_model = "vgg16"
+model_folder = "model"
+archive_file = "{}.tar.gz".format(current_model)
+archive_path = os.path.join(model_folder, archive_file)
+url = "{}{}".format(base_url, archive_file)
+onnx_path = os.path.join(model_folder, current_model, 'model.onnx')
+
+# Create the model folder and download the zipped model
+if not os.path.isdir(model_folder):
+    os.makedirs(model_folder)
+if not os.path.isfile(archive_path):
+    print('Downloading the {} model to {}...'.format(current_model, archive_path))
+    wget.download(url, model_folder)
+    print('{} downloaded'.format(current_model))
+
+# Extract the model
+if not os.path.isdir(os.path.join(model_folder, current_model)):
+    print('Extracting {} in {}...'.format(archive_path, model_folder))
+    tar = tarfile.open(archive_path, "r:gz")
+    tar.extractall(model_folder)
+    tar.close()
+    print('Model extracted.')
+```
+
+## Downloading the Caltech101 dataset
+
+The [Caltech101 dataset](http://www.vision.caltech.edu/Image_Datasets/Caltech101/) is made of pictures of objects belonging to 101 categories. About 40 to 800 images per category. Most categories have about 50 images.
+
+*L. Fei-Fei, R. Fergus and P. Perona. Learning generative visual models from few training examples: an incremental Bayesian approach tested on 101 object categories. IEEE. CVPR 2004, Workshop on Generative-Model
+Based Vision. 2004*
+
+
+```python
+data_folder = "data"
+dataset_name = "101_ObjectCategories"
+archive_file = "{}.tar.gz".format(dataset_name)
+archive_path = os.path.join(data_folder, archive_file)
+data_url = "https://s3.us-east-2.amazonaws.com/mxnet-public/"
+if not os.path.isdir(data_folder):
+    os.makedirs(data_folder)
+if not os.path.isfile(archive_path):
+    print('Downloading {} in {}...'.format(archive_file, data_folder))
+    wget.download("{}{}".format(data_url, archive_file), data_folder)
+    print('Extracting {} in {}...'.format(archive_file, data_folder))
+    tar = tarfile.open(archive_path, "r:gz")
+    tar.extractall(data_folder)
+    tar.close()
+    print('Data extracted.')
+```
```

Comment:
Should we add this to mxnet.gluon.data.vision.datasets as another option? So that other tutorials can also benefit from this.

Reply:
At the moment there is no big image shape dataset inbuilt. This is also required for the tutorial I'm working on. Cifar/Mnist are too small for my use-case.

Reply:
yes I had the same issue that's why I used Caltech101. That's a good idea, we'd probably need to check the license first.

Reply:
Also, It might be better to rename the folder to say Caltech101 and not just 101_ObjectCategories in the s3 bucket?